src/OpenDiffix.Core/Anonymizer.fs

```diff
@@ -306,5 +308,6 @@ let count (executionContext: ExecutionContext) (perAidContributions: AidCountSta
     byAid
     |> Array.choose id
     |> anonymizedSum
-    |> Option.map (Math.roundAwayFromZero >> int64 >> Result.Count)
-    |> Option.defaultValue NotEnoughAIDVs
+    |> Math.roundAwayFromZero
+    |> int64
+    |> Result.Count
```

Comment:
This option stuff goes away because we do the check in `CountDistinct.Final`, right?

Reply:
yes, thanks to this check (and the existing one in `DiffixCount.Final`), we can make the assumption that it's a non-empty list. With that assumption `anonymizedSum` becomes optionless. See also comment near `anonymizedSum`